Postgres/AddServer.swift

```diff
@@ -60,7 +79,16 @@ class AddServerViewController: NSViewController, MainWindowModelConsumer {
 	
 	@IBAction func createServer(_ sender: AnyObject?) {
 		guard self.view.window!.makeFirstResponder(nil) else { NSSound.beep(); return }
-        guard availableBinaries.indices.contains(selectedVersionIdx) else {
+        
+        let isValid = if availableBinaries.indices.contains(selectedVersionIdx) {
```

Comment:
This check isn't necessary anymore, since the array is no longer accessed in this function.